strongbox-db-liquibase/src/main/resources/db/changelog/v1.0.0/v1.0.0.21__ArtifactEntry.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.21" author="strongbox-dev@carlspring.com">
+
+        <o:changePropertyTypeDateToDatetime propertyName="created" className="ArtifactEntry"/>
```

Comment:
as commented in https://github.com/strongbox/liquibase-orientdb/pull/2, `propertyName` doesn't match your schema

Reply:
@fuss86 in fact all this complexity was not needed. Simply dropping meta info about 'created' property and recreating it again with a new data type was enough. So we don't need any changes into liquibase-orientdb

Reply:
Will it update previous values to date with time ?

Reply:
@fuss86, previous values don't have time information. The moment the java.util.Date hit the database with 'created' property of type date, time part of it was lost. 
By the time we apply the migration, there's no time information in orientDb.